cli/commands/site/create.ts

```diff
@@ -0,0 +1,331 @@
+					await generateSiteCertificate( siteDetails.customDomain );
+					logger.reportSuccess( __( 'SSL certificates generated' ) );
+				}
+
+				logger.reportStart(
+					LoggerAction.ADD_DOMAIN_TO_HOSTS,
+					__( 'Adding domain to hosts file...' )
+				);
+				try {
+					await addDomainToHosts( siteDetails.customDomain, siteDetails.port );
+					logger.reportSuccess( __( 'Domain added to hosts file' ) );
+				} catch ( error ) {
+					throw new LoggerError( __( 'Failed to add domain to hosts file' ), error );
+				}
+			}
+
+			logger.reportStart( LoggerAction.START_SITE, __( 'Starting WordPress site...' ) );
+			try {
+				await startWordPressServer( siteDetails, { wpVersion, blueprint } );
+				logger.reportSuccess( __( 'WordPress site started' ) );
+
+				logSiteDetails( siteDetails );
+				await openSiteInBrowser( siteDetails );
+			} catch ( error ) {
+				throw new LoggerError( __( 'Failed to start WordPress server' ), error );
+			}
+		} else if ( blueprint ) {
+			logger.reportStart( LoggerAction.START_DAEMON, __( 'Starting process daemon...' ) );
+			await connect();
+			logger.reportSuccess( __( 'Process daemon started' ) );
+
+			logger.reportStart( LoggerAction.START_SITE, __( 'Applying blueprint...' ) );
+			try {
+				await runBlueprint( siteDetails, { wpVersion, blueprint } );
+				logger.reportSuccess( __( 'Blueprint applied successfully' ) );
+			} catch ( error ) {
+				throw new LoggerError( __( 'Failed to apply blueprint' ), error );
+			}
+
+			console.log( __( '\nSite created successfully!\n' ) );
+			logSiteDetails( siteDetails );
+			console.log( __( '\nRun "studio site start" to start the site.' ) );
+		} else {
+			console.log( __( '\nSite created successfully!\n' ) );
+			logSiteDetails( siteDetails );
+			console.log( __( '\nRun "studio site start" to start the site.' ) );
+		}
+	} catch ( error ) {
+		if ( error instanceof LoggerError ) {
+			logger.reportError( error );
+		} else {
+			const loggerError = new LoggerError( __( 'Failed to create site' ), error );
+			logger.reportError( loggerError );
+		}
+		process.exit( 1 );
+	} finally {
+		disconnect();
+	}
+}
+
+export const registerCommand = ( yargs: StudioArgv ) => {
+	return yargs.command( {
+		command: 'create',
+		describe: __( 'Create a new local site' ),
+		builder: ( yargs ) => {
+			return yargs
+				.option( 'name', {
+					type: 'string',
+					describe: __( 'Site name' ),
+				} )
+				.option( 'wp-version', {
+					type: 'string',
+					describe: __( 'WordPress version (e.g., "latest", "6.4", "6.4.1")' ),
+					default: DEFAULT_WORDPRESS_VERSION,
+				} )
+				.option( 'php-version', {
+					type: 'string',
+					describe: __( 'PHP version' ),
+					choices: ALLOWED_PHP_VERSIONS,
+					default: DEFAULT_PHP_VERSION,
+				} )
+				.option( 'custom-domain', {
+					type: 'string',
+					describe: __( 'Custom domain (e.g., "mysite.local")' ),
+				} )
+				.option( 'enable-https', {
```

Comment:
```suggestion
				.option( 'https', {
```
Same thing about brevity.